mobile/lib/modules/partner/views/partner_detail_page.dart

```diff
@@ -14,6 +15,14 @@ class PartnerDetailPage extends HookConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     final assets = ref.watch(assetsProvider(partner.isarId));
 
+    useEffect(
+      () {
+        ref.read(assetProvider.notifier).getPartnerAssets(partner);
```

Comment:
What is the need of having this method invoked here?

Reply:
Why didn't we use this before?

Reply:
I don't know. :-D We didn't have the getPartnerAssets method for a specific user.